setup/command/list.go

```diff
@@ -18,23 +18,23 @@ func (f *CommandFactory) List() cli.Command {
 		Name:  "list",
 		Usage: "list local and/or remote Layer0 instances",
 		Flags: append(awsFlags,
-			cli.BoolTFlag{
-				Name:  "l, local",
-				Usage: "show local Layer0 instances, denoted by 'l' (default: true)",
+			cli.BoolFlag{
```

Comment:
There is now an inconsistency with these changes, as you've changed this to `BoolFlag` but then didn't change the flag on [line 25](https://github.com/quintilesims/layer0/pull/430/commits/510b2456bb73f813a6dc8980087140979be22aac#diff-1f61dfa67bf4a780befc9cfc3c65388aR25) to be consistent.

Also, this is still a stylistic thing, but this still feels really awkward to me. Compare what we're doing here to what `git` does when you list branches.

`l0-setup` would show local and remote l0 instances
`l0-setup --no-local` shows just remote
`l0-setup --no-remote shows just local`

`git branch` shows you your local branches
`git branch -r` shows you just remote branches
`git branch -a` shows you local and remote branches

This git model feels much cleaner and more intuitive to me. Is there a specific reason we're designing list to behave this way?

Reply:
we don't have to deal with conflicting flags in my pattern